### PR TITLE
Designer 14 update

### PR DESCRIPTION
**Pull request by skjnldsv**
Update for nextcloud 14
Please help @juliushaertl :)

@nextcloud/designers @nextcloud/documentation 

- [x] css mixins
- [x] css vars
- [x] new app-navigation button
- [x] list design update
- [x] new layout
- [ ] search
- [ ] ...???

### CODE REVIEW

**Comment by juliushaertl** [comment]
Looks good. I've pushed a small section regarding the css4 variables. @skjnldsv I think we should keep the search docs separate from the design chapter, since it is not really design stuff but a frontend api.

**Comment by MorrisJobke** [comment]
Can we merge this for now and continue with followup PRs?

**Comment by skjnldsv** [comment]
@MorrisJobke can't we wait next week? :)

**Comment by MorrisJobke** [comment]
> @MorrisJobke can't we wait next week? :)

Was just a question ;)

**Comment by skjnldsv** [comment]
@nextcloud/designers LET'S GET THIS IN!
@juliushaertl for the search, maybe we should use the API manual https://docs.nextcloud.com/server/14/developer_manual/api.html

And start listing our open APIs with javascript as well?
`OCA.Search`... stuff like that?

**Comment by juliushaertl** [comment]
> @juliushaertl for the search, maybe we should use the API manual 
 docs.nextcloud.com/server/14/developer_manual/api.html
> And start listing our open APIs with javascript as well?
> OCA.Search... stuff like that?

:+1: Yes, should be not that hard with jsdoc and some jsdoc to restructuredText converter.

**Comment by juliushaertl** [comment]
@skjnldsv There seems to be an error with the images:

```
[156]

! LaTeX Error: File `{design/img/actions/add*}' not found.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.10190 ...hinxpxdimen]{{design/img/actions/add*}}
                                                  
? 
! Emergency stop.
 ...                                              
                                                  
l.10190 ...hinxpxdimen]{{design/img/actions/add*}}
```

File: README.rst [hunk 1]
```diff
@@ -162,5 +162,13 @@ Process
 3. Move the resulting ``rst`` files in place and reference them
 4. Wrap text lines at 80 chars, apply markup fixes
 
+Icons
+^^^^^^^
```
Comment (nickvergessen) [inline README.rst]: too long

**Comment by skjnldsv** [comment]
@nickvergessen I removed the designer pdf build.
The pdf is unusable in the current state anyway, the php api is broken, the icons list take like 3 pages, images are on a page and have then 80% blank space. I'd say it's fien to remove it imho.

What does @nextcloud/designers think?
@jancborchardt

**Comment by skjnldsv** [comment]
@nickvergessen what do we do here?

**Comment by jancborchardt** [comment]
Looks good to me, what’s holding it up?

**Comment by skjnldsv** [comment]
@jancborchardt the removal of the pdf tests. Can you talk to Morris about this? :)

**Comment by MorrisJobke** [comment]
> @jancborchardt the removal of the pdf tests. Can you talk to Morris about this? :)

What do you mean by that? PDFs are not generated anymore since 12 or 13 AFAIK.

**Comment by skjnldsv** [comment]
@MorrisJobke but drone tests for off builds still are :)

**Comment by skjnldsv** [comment]
@nextcloud/designers please review and merge, this is becoming urgent!

File: Makefile [hunk 2]
```diff
@@ -35,10 +35,6 @@ user-manual-de-pdf:
 	cd user_manual_de && make latexpdf
 	@echo "User manual de build finished; PDF is updated"
 
-developer-manual-pdf:
-	cd developer_manual && make latexpdf
-	@echo "Developer manual build finished; PDF is updated"
-
```
Comment (MorrisJobke) [inline Makefile]: Nice one 👍

Comment (MorrisJobke) [reply Makefile]: 🙈

**Comment by MorrisJobke** [comment]
Cherry-picked to stable14 - last commit is b3ed45f2

**Pull request merged**